resources/contributors.txt

```diff
@@ -65,6 +65,7 @@ ZonianMidian | https://github.com/ZonianMidian | :/avatars/zonianmidian.png | Co
 olafyang | https://github.com/olafyang | | Contributor
 chrrs | https://github.com/chrrs | | Contributor
 4rneee | https://github.com/4rneee | | Contributor
+sheybey | https://github.com/sheybey | | Contributor
```

Comment:
separate PR

Reply:
ok, I will remove this commit when I rebase on current master